src/main/wallet-connect/wallet-connect-service.js

```diff
@@ -130,7 +131,9 @@ export class WalletConnectService {
 		rawTx.nonce = await this.web3Service.getNextNonce(rawTx.from);
 		const tx = { ...rawTx };
 		tx.gas = EthUtils.hexToDecimal(tx.gas);
-		tx.gasPrice = EthUtils.hexToDecimal(tx.gasPrice);
+		// TODO: Workaround for gasPrice, ideally we should not override gasPrice
+		const gasStationInfo = await this.ethGasStationService.getInfo();
+		tx.gasPrice = gasStationInfo.avarage;
```

Comment:
Typo, should be ethGasStationInfo.average

Reply:
Yeah, this is wrong. I've pushed the updated version